terraform/deployments/cluster-services/variables.tf

```diff
@@ -21,9 +28,11 @@ variable "govuk_environment" {
 variable "dex_github_orgs_teams" {
   type        = list(object({ name = string, teams = list(string) }))
   description = "List of GitHub orgs and associated teams that Dex authorises. Format [{name='github_org', teams=['github_team_name']}] "
+  default     = [{ name = "alphagov", teams = ["gov-uk-production"] }]
```

Comment:
is it missing teams = gov-uk?

Reply:
Oh, good catch! So this particular line is right, but I'd deleted the overrides in test and integration by mistake. Fixed. Thanks!